core/org.openjdk.jmc.rjmx.common/pom.xml

```diff
@@ -35,12 +35,28 @@
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.openjdk.jmc</groupId>
-		<artifactId>missioncontrol.application</artifactId>
+		<artifactId>missioncontrol.core</artifactId>
 		<version>${revision}${changelist}</version>
 	</parent>
-	<artifactId>org.openjdk.jmc.rjmx.services.jfr</artifactId>
-	<packaging>eclipse-plugin</packaging>
+	<artifactId>rjmx.common</artifactId>
 	<properties>
 		<jmc.config.path>${project.basedir}/../../configuration</jmc.config.path>
 	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.openjdk.jmc</groupId>
+			<artifactId>common</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.openjdk.jmc</groupId>
+			<artifactId>flightrecorder</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+    	<dependency>
```

Comment:
Indentation in this line is still wrong.

Reply:
Ah, thank you now I see it.